app/src/cli/util.ts

```diff
@@ -0,0 +1,71 @@
+import * as Path from 'path'
+
+import stripAnsi = require('strip-ansi')
```

Comment:
I think we can switch this over to `import * as stripAnsi from 'strip-ansi'`, right?